### DISCUSSION TITLE
Curso Fullstack Master - DevPleno

### DISCUSSION THREAD

**Discussion by brunoolympioferreira** [frontendbr/forum#2114] [Perguntas e Respostas]
Olá pessoal
Alguém aqui já fez o curso de Fullstack do Tulio ? Se sim indiciaria o investimento ? Conheço bem lógica de programação e algoritmos mas ainda nao entrei em nenhuma linguagem front-end, haveria alguma dificuldade nisso?

**Comment by theandersonn** [comment]
@brunoolympioferreira Fiz parte do curso `devReact`, as explicações, didática são boas, a estrutura do curso precisa ser organizada melhor, no geral eh um bom curso.

Pensando em FullStack uma boa e ir por partes, front ou back, porque se não virá uma enxurrada de informações, difícil de assimilar tanta coisa.

Por exemplo: 
Front - HTML, CSS, JavaScript...
Back - Banco de dados [relacional, não relacional], protocolo HTTP, Node.js...
